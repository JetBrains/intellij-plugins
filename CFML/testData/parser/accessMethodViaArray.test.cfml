<cfscript>
  variables.mycfc[methodName](arguments);
</cfscript>