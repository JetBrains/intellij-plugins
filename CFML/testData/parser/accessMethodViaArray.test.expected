CfmlFile:accessMethodViaArray.cfml
  Tag
    PsiElement(OPENER)('<')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(R_ANGLEBRACKET)('>')
    FunctionCallExpression
      None
        ReferenceExpression
          PsiElement(SCOPE_KEYWORD)('variables')
          PsiElement(POINT)('.')
          PsiElement(IDENTIFIER)('mycfc')
        PsiElement(L_SQUAREBRACKET)('[')
        ReferenceExpression
          PsiElement(IDENTIFIER)('methodName')
        PsiElement(R_SQUAREBRACKET)(']')
      ArgumentList
        PsiElement(L_BRACKET)('(')
        Argument arguments
          PsiElement(IDENTIFIER)('arguments')
        PsiElement(R_BRACKET)(')')
    PsiElement(SEMICOLON)(';')
    PsiElement(LSLASH_ANGLEBRACKET)('</')
    PsiElement(CF_TAG_NAME)('cfscript')
    PsiElement(CLOSER)('>')